eng/Versions.props

```diff
@@ -80,6 +80,29 @@
     <MicrosoftFileFormatsVersion>1.0.120601</MicrosoftFileFormatsVersion>
     <!-- Libraries dependencies -->
     <runtimenativeSystemIOPortsVersion>5.0.0-alpha.1.19563.3</runtimenativeSystemIOPortsVersion>
+    <SystemBuffersVersion>4.5.0</SystemBuffersVersion>
+    <SystemCollectionsVersion>4.3.0</SystemCollectionsVersion>
+    <SystemCollectionsConcurrentVersion>4.3.0</SystemCollectionsConcurrentVersion>
+    <SystemComponentModelAnnotationsVersion>4.7.0</SystemComponentModelAnnotationsVersion>
+    <SystemDiagnosticsContractsVersion>4.3.0</SystemDiagnosticsContractsVersion>
+    <SystemDiagnosticsDebugVersion>4.3.0</SystemDiagnosticsDebugVersion>
+    <SystemDiagnosticsTracingVersion>4.3.0</SystemDiagnosticsTracingVersion>
+    <SystemDynamicRuntimeVersion>4.3.0</SystemDynamicRuntimeVersion>
+    <SystemMemoryVersion>4.5.3</SystemMemoryVersion>
```

Comment:
Looks like these aren't the newest versions /cc @joperezr

Reply:
That's correct, they aren't. How did you get these reference numbers @ViktorHofer? From looking at Darc, it seems like release/2.1 branch in corefx (where System.Memory is built and shipped) doesn't have a default channel any longer, which would explain why we didn't get the new one if we used dependency flow. I would suggest two things: one is to update that channel to have all serviced package versions from 2.1, and the other one is to run my validation task somewhere in the build to make sure that these package versions are still the latest.